### ISSUE TITLE
summation((n+1) / (n*(2*n-1)*(2*n+1)), [n, 2, oo]) returns nan

### ISSUE THREAD

**Issue by ethankward** [sympy/sympy#15838]
```
>>> from sympy import *
>>> n = Symbol('n')
>>> summation((n+1) / (n*(2*n-1)*(2*n+1)), [n, 2, oo])
nan
```

The actual value has a simple form: `-7/6 + log(4)`.

The finite summation also gives `nan`:
```
>>> k = Symbol('k')
>>> summation((n+1) / (n*(2*n-1)*(2*n+1)), [n, 2, k])
nan
```

**Comment by RituRajSingh878** [comment]
I am working on this.

**Comment by RituRajSingh878** [comment]
I think the problem is, it is unable to calculate this summation so it is returning nan.

**Comment by jmig5776** [comment]
@ethankward Can you tell me from which series `(n+1) / (n*(2*n-1)*(2*n+1)` it is evaluating to `-7/6 + log(4)` so that i could workout to get this solve in sympy?

**Comment by ethankward** [comment]
I don't understand your question. That's the symbolic value of the summation which you can verify using other computer algebra systems or by hand.

**Comment by jmig5776** [comment]
Okk I got my answer I am trying to solve it. And I think this should not return nan, and sympy should able to solve it.

**Comment by RituRajSingh878** [comment]
> Okk I got my answer I am trying to solve it. And I think this should not return nan, and sympy should able to solve it.

@jmig5776 I am working and I already made a pr. So you can go for other issues.

**Comment by jmig5776** [comment]
@RituRajSingh878 There is nothing wrong with more than one person doing a PR for the same issue. Sometimes that will help everyone find the best solution.

**Comment by RituRajSingh878** [comment]
yup But You can look my pr and can give some suggestions it would be better for both of Us.
and save the time of both people.

**Comment by ethren** [comment]
@jmig5776 
Sorry I think you shouldn't belong to Sympy you are useless. Please something else that could be convenient for you. Sympy doesn't work by hardcoding.

**Comment by wuyudi** [comment]
```python
summation(1/(i*(2*i+1)),(i,1,oo))
```

this also return NaN, but answer is [2-2*log(2)](https://math.stackexchange.com/questions/1334870/how-to-find-sum-of-the-infinite-series-sum-n-1-infty-frac1-n2n1)
